deploy/charts/harvester/templates/deployment.yaml

```diff
@@ -168,3 +185,27 @@ spec:
         ports:
         - name: https
           containerPort: {{ .Values.webhook.httpsPort }}
+{{- if .Values.enableGoCoverDir }}
+        volumeMounts:
+        - name: go-cover-dir
+          mountPath: /coverdatafiles
+      volumes:
+      - name: go-cover-dir
+        persistentVolumeClaim:
+          claimName: go-cover-dir-pvc
+{{- end }}
+---
+{{- if .Values.enableGoCoverDir }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: go-cover-dir-pvc
+  namespace: harvester-system
```

Comment:
From https://go.dev/testing/coverage/, we need a folder for Golang to write coverage files, so I add a PVC when `enableGoCoverDir` is `true`.

Reply:
Replace the Longhorn PVC with Host path PVC, so we don't need to rely on LH components.